### PR TITLE
Tools: add  fpga smart nic n6010 board module

### PR DESCRIPTION
**Pull request by anandaravuri**
- add  board_n6010 module to read and print board spec information
     like firmware version, phy (ethernet), mac, security key etc.

Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: tools/libboard/board_n6010/board_n6010.c [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+#include <stdlib.h>
+
+#include "../board_common/board_common.h"
+#include "board_n6010.h"
+
+#define UNUSED_PARAM(x) ((void)x)
+
+// print mac information
+fpga_result print_mac_info(fpga_token token)
+{
```
Comment (tswhison) [inline tools/libboard/board_n6010/board_n6010.c:53]: Will these be implemented in this PR or in a different one?

Comment (anandaravuri) [reply tools/libboard/board_n6010/board_n6010.c:53]: Implementation in different PR,  MAC, PHY, MAX10/NIOS version sysfs attributes and paths  are going to change FPGA  driver for N6010 smart nic card. place holder for implementation .

Comment (tswhison) [reply tools/libboard/board_n6010/board_n6010.c:53]: Ok.  Please update the title to include an indication that this is just a placeholder intended to be a WIP.  The title is misleading as currently worded.

File: tools/libboard/board_n6010/board_n6010.c [hunk 2]
```diff
@@ -0,0 +1,271 @@
+// Copyright(c) 2021, Intel Corporation
```
Comment (tswhison) [inline tools/libboard/board_n6010/board_n6010.c:1]: Unit tests for this new library?

**Review by michelleyho — DISMISSED** [review]

**Review by michelleyho — DISMISSED** [review]

**Review by michelleyho — APPROVED** [review]

File: tools/libboard/board_n6010/board_n6010.c (hunk 2 above)
Comment (anandaravuri) [reply tools/libboard/board_n6010/board_n6010.c:1]: added unit test cases

**Review by tswhison — APPROVED** [review]

**Pull request merged**